### ISSUE TITLE
A probable issue with Interactive Latent (Upscale) node.

### ISSUE THREAD

**Issue by mart-hill** [ltdrdata/ComfyUI-Impact-Pack#349]
I've noticed a weird UI behavior after todays updates - the UI freezes after 'reaching' the mentioned node, which has never happened before:
![image](https://github.com/comfyanonymous/ComfyUI/assets/55754730/e44c88b2-72c3-4cca-ab91-cc74e1ee1c2e)
[woodland.json](https://github.com/comfyanonymous/ComfyUI/files/13620971/woodland.json)

Before I execute it, the workflow looks like this:
![image](https://github.com/comfyanonymous/ComfyUI/assets/55754730/a09dd873-5ad9-4568-a5f6-46be1520cfdd)

![image](https://github.com/comfyanonymous/ComfyUI/assets/55754730/46d783ef-ea5a-4364-939d-618442fa630f)

The only thing I can move is the main menu bar, everything else is frozen, until I refresh the page.

F12 shows:
```
Uncaught TypeError: ctx.save is not a function
    at nodeType.onDrawForeground (impact-pack.js:65:8)
    at drawBadge (comfyui-manager.js:283:25)
    at node.onDrawForeground (comfyui-manager.js:955:45)
    at LGraphCanvas.drawNode (litegraph.core.js:8518:18)
    at LGraphCanvas.drawNode (snapToGrid.js:86:24)
    at LGraphCanvas.drawNode (app.js:1146:29)
    at LGraphCanvas.drawNode (snapToGrid.js:32:21)
    at LGraphCanvas.drawFrontCanvas (litegraph.core.js:7801:22)
    at LGraphCanvas.draw (litegraph.core.js:7716:18)
    at LGraphCanvas.renderFrame (litegraph.core.js:5783:22)
toolbarContentscript.js:113 {"message":"A listener indicated an asynchronous response by returning true, but the message channel closed before a response was received"}

```
Despite this issue, the image generation finishes successfully, and there are no errors in Powershell window. 🙂 

ComfyUI: 1795 [a4ec54] (2023-12-08) and ComfyUI: 1795 [97015b] (2023-12-08) commits - that's when I noticed this issue. :)
Manager: V1.10

**Comment by ltdrdata** [comment]
oh..it's a bug related to Manager. I need to fix it.

**Comment by ltdrdata** [comment]
This is fixed on ComfyUI-Manager V1.10.3.
Update ComfyUI-Manager.
